src/elarm_alarmlist.erl

```diff
@@ -151,36 +151,36 @@ al_test_() ->
      ]}.
 
 just_started() ->
-    {ok,State} = init([]),
-    ?assertEqual({{ok,[]}, State}, get_alarms(State)),
-    EvtId = erlang:now(),
-    ?assertEqual({{error, not_active},State}, get_alarm(EvtId, State)),
-    ?assertEqual({{error, not_active},State}, get_alarm(full, disk_1, State)).
+    {ok, State} = init([]),
+    ?assertEqual({{ok, []}, State}, get_alarms(State)),
+    EvtId = erlang:timestamp(),
```

Comment:
@hcs42 how do you plan to support OTP 18.x, then? Will you do something like…
```erlang
case erlang:function_exported(erlang, timestamp, 0) of
  true -> erlang:timestamp();
  false -> erlang:now()
end
```

Maybe you should consider switching to uuids.